armbian/build.sh

```diff
@@ -7,6 +7,13 @@
 #
 set -eu
 
+# Settings
+#
+# VirtualBox Number of CPU cores
+VIRTUALBOX_CPU="4"
```

Comment:
(Optional) We could also take the environment variable from the caller, and setting it to the current value in case it's unspecified:

```
VIRTUALBOX_CPU=${VIRTUALBOX_CPU:-"4"}
```

That way, an invocation like `VIRTUALBOX_CPU=8 bash build.sh` would e.g use 8 cores.

(Same suggestion for `VIRTUALBOX_MEMORY`.)